trie/utils/verkle_test.go

```diff
@@ -34,13 +35,19 @@ func TestGetTreeKey(t *testing.T) {
 	n := uint256.NewInt(1)
 	n = n.Lsh(n, 129)
 	n.Add(n, uint256.NewInt(3))
-	GetTreeKey(addr[:], n, 1)
+	tk := GetTreeKey(addr[:], n, 1)
+
+	got := hex.EncodeToString(tk)
+	exp := "f42f932f43faf5d14b292b9009c45c28da61dbf66e20dbedc2e02dfd64ff5a01"
+	if got != exp {
+		t.Fatalf("Generated trie key is incorrect: %s != %s", got, exp)
+	}
```

Comment:
I took the liberty to improve a bit on this test, and to enforce an expected value.

I did this in the first commit of the PR before changing `GetTreeKey(...)` so I could have an anchor source of truth to know if the improved implementation was generating the same output as before. The branch generates the `f42f9...` value before the improvement.

This test still passing makes me more comfortable about the change (and future ones, if any). I don't know if you have some way to gain more confidence in some other branch or similar; it never hurts. :)

Reply:
yeah that makes sense, thank you for that.